plugins/cilium-docker/driver/driver.go

```diff
@@ -77,8 +77,16 @@ func newLibnetworkRoute(route plugins.Route) api.StaticRoute {
 	return rt
 }
 
-// NewDriver creates and returns a new Driver for the given API URL
+// NewDriver creates and returns a new Driver for the given API URL.
+// If url is nil then use SockPath provided by CILIUM_SOCK
+// or the cilium default SockPath
+
```

Comment:
The empty line detaches this comment from the function below (so it won't show up in godocs). Was this intentional? (probably not?)

Reply:
oops sorry I fixed it, wasn't intentional.

Reply:
no worries :)